force-app/main/default/aura/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+    "plugins": ["@salesforce/eslint-plugin-aura"],
+    "extends": [
+        "plugin:@salesforce/eslint-plugin-aura/recommended",
+        "prettier"
+    ],
+    "rules": {
+        "func-names": "off",
```

Comment:
Why are these rules in here?

Reply:
These Aura rules are disabled here to avoid a large rewrite of the Aura code and to keep consistency with the other sample apps.